.github/workflows/data_update_v2.yml

```diff
@@ -0,0 +1,73 @@
+name: data.v2.json update
+
+on:
+  schedule:
+    # Run at 3 AM PDT every day
+    - cron: 0 10 * * *
+
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checkout the main repo
+    - name: Checkout Main repo
+      uses: actions/checkout@v2
+      with:
+        ssh-key: ${{ secrets.data_daily_update }}
+        path: site
+
+    # Checkout data scraper repo
+    - name: Checkout Data Scraper
+      uses: actions/checkout@v2
+      with:
+        ssh-key: ${{ secrets.data_daily_update }}
+        repository: sfbrigade/data-covid19-sfbayarea
+        path: scraper
+
+    # The scraper uses Python 3.7+, so make sure we've got the latest 3.x
+    - name: Set up Python 3.x
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+
+    # Install dependencis
+    # - The commit that was checked out will be available as $SCRAPER_COMMIT.
+    - name: Install Data Scraper & Dependencies
+      run: |
+        cd ${GITHUB_WORKSPACE}/scraper
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt;
+
+        # Keep track of the version used so we can use it in commit messages
+        echo "::set-env name=SCRAPER_COMMIT::$(git rev-parse HEAD)"
+
+    - name: Scrape Data
+      run: |
+        echo "::set-env name=SCRAPER_TIME::$(date)"
+        cd ${GITHUB_WORKSPACE}/scraper
+        python scraper_data.py > "${GITHUB_WORKSPACE}/site/data/data.v2.json"
```

Comment:
This is the only line (besides the workflow name) that is different from the existing data workflow.